theme/components/stickytoolbar.scss

```diff
@@ -5,14 +5,12 @@
 	.ck-toolbar.ck-toolbar_sticky {
 		@include ck-drop-shadow();
 
-		position: fixed;
 		top: 0;
```

Comment:
Maybe the best way to decide what is the best place for them is the question: "If this element will look OK without this properties." If not then should be placed in the Wireframe.